overlayer/textractoverlayer/t_overlay.py

```diff
@@ -20,11 +20,20 @@ def doc_height(self):
 
 
 @dataclass
-class BoundingBox():
```

Comment:
Can you use the style.yapf to format the code. It has column_limit: 120 and would make reading the changes easier.